.github/workflows/check-pr-title.yaml

```
@@ -0,0 +1,39 @@
+name: Check PR Title
+
+on:
+  pull_request_target:
+    types:
+      - edited
+      - opened
+      - reopened
+      - synchronize
+
```

We could add concurrency here to not run this gh action everytime someone updates the pr title:
```suggestion
concurrency:
  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.event.number }}
  cancel-in-progress: true
  
```
FYI notebooks do the same https://github.com/kubeflow/notebooks/blob/861cd579ea795f252c0ac36a55bcd52988d0692e/.github/workflows/semantic-prs.yaml#L11-L13